Containerfile

```diff
@@ -20,8 +20,21 @@ RUN chown -R nginx:nginx /usr/share/nginx/html /etc/nginx/certs /var/log/nginx /
 
 COPY nginx/ /etc/nginx/
 
+RUN mkdir -p /licenses
+COPY LICENSE /licenses
```

Comment:
We can add this license to the upstream Containerfile as well.

Reply:
Only one Containerfile now.